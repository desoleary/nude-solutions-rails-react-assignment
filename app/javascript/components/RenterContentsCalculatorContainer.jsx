import React from 'react';

const RenterContentsCalculatorContainer = () => (
  <>Display List of Renter's Contents</>
);

export default RenterContentsCalculatorContainer;
