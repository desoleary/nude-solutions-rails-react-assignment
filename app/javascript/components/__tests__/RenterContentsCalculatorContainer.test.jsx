import { mount } from 'enzyme';
import React from 'react';
import RenterContentsCalculatorContainer from '../RenterContentsCalculatorContainer';

describe('RenterContentsCalculatorContainer', () => {
  describe('with shallow rendering', () => {
    let wrapper;

    beforeEach(() => {
      wrapper = mount(<RenterContentsCalculatorContainer />);
    });

    it('compares snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
