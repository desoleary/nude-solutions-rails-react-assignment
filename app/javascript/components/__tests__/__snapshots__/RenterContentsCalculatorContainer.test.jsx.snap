// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RenterContentsCalculatorContainer with shallow rendering compares snapshot 1`] = `
<RenterContentsCalculatorContainer>
  Display List of Renter's Contents
</RenterContentsCalculatorContainer>
`;
